control_plane/src/bin/neon_local.rs

```diff
@@ -719,19 +735,36 @@ fn handle_timeline(timeline_match: &ArgMatches, env: &mut local_env::LocalEnv) -
             println!("Importing timeline into pageserver ...");
             pageserver.timeline_import(tenant_id, timeline_id, base, pg_wal, pg_version)?;
             println!("Creating node for imported timeline ...");
-            env.register_branch_mapping(name.to_string(), tenant_id, timeline_id)?;
-            cplane.new_node(tenant_id, name, timeline_id, None, None, pg_version, None)?;
+            env.register_branch_mapping(
+                name.to_string(),
+                tenant_id,
+                timeline_id,
+                RegionId::default(),
+            )?;
+            cplane.new_node(
+                tenant_id,
+                name,
+                timeline_id,
+                None,
+                None,
+                pg_version,
+                RegionId::default(),
+            )?;
             println!("Done");
         }
         Some(("branch", branch_match)) => {
             let tenant_id = get_tenant_id(branch_match, env)?;
             let new_branch_name = branch_match
                 .value_of("branch-name")
                 .ok_or_else(|| anyhow!("No branch name provided"))?;
+            let region_id = branch_match
```

Comment:
We now use `--region-id` to associate a newly created timeline with a region. E.g.
```
neon timeline branch --branch-name r1 --region-id 1
```